server/cli/cli_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter.mattis@gmail.com)
+
+package cli
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"code.google.com/p/go-commander"
+	"github.com/cockroachdb/cockroach/server"
+)
+
+type cliTest struct {
+	*server.TestServer
+	commander.Commander
+}
+
+func newCLITest() cliTest {
+	osExit = func(int) {}
+	osStderr = os.Stdout
+
+	s := &server.TestServer{}
+	if err := s.Start(); err != nil {
+		log.Fatalf("Could not start server: %v", err)
+	}
+
+	c := commander.Commander{
+		Name: "cockroach",
+		Commands: []*commander.Command{
```

Comment:
So we have to keep this in sync with the one in our main application? Is it just a duplicate? Perhaps we could put the commander itself somewhere central and just call in from this test.